pyproject.toml

```diff
@@ -0,0 +1,38 @@
+[tool.poetry]
+name = "klaam"
+version = "0.0.1"
+description = "Arabic speech recognition, classification and text-to-speech."
+license = "MIT"
+authors = [
+    "Zaid Alyafeai <alyafey22@gmail.com>",
+    "mustafa0x"
+]
+readme = 'README.md'
+
+[tool.poetry.dependencies]
+python = "^3.8"
+rich = "^12.2.0"
```

Comment:
I do not think we are using _rich_ in klaam here, are we?

Reply:
I just included rich because it makes the output nicer :p there isn't any harm for that. I can remove it.